src/containers/ReportArticle/state/types.js

```diff
@@ -0,0 +1,7 @@
+export const REPORTING_SUCCESS = 'REPORTING_SUCCESS';
+export const REPORTING_FAILURE = 'REPORTING_FAILURE';
+export const FETCH_REPORT_TYPES = 'FETCH_REPORT_TYPES';
+export const FETCH_REPORT_TYPES_SUCCESS = 'FETCH_REPORT_TYPES_SUCCESS';
+export const FETCH_REPORT_TYPES_FAILURE = 'FETCH_REPORT_TYPES_FAILURE';
+export const REPORT_MORE_THAN_ONCE = 'You cannot report this article more than once!';
+export const REPORT_WITHOUT_DATA = 'All fields (Violation Type and Description) are required!';
```

Comment:
@SrMoffat I believe this file should only contain `redux action type` definitions. You can move this constant to where it's being used.

Reply:
You make a valid point here. I have removed the `REPORT_WITHOUT_DATA` and `REPORT_MORE_THAN_ONCE` and placed them in another files called `messages`